app/controllers/restaurants_controller.rb

```diff
@@ -0,0 +1,11 @@
+class RestaurantsController < ApplicationController
+  def index
+    restaurants = Restaurant.all
+    render json: restaurants.as_json(include: [:addresses, :reviews])
```

Comment:
Nice, but you probably want to use a serializer https://github.com/Netflix/fast_jsonapi